sources/meta-srobo/recipes-extended/sudo/sudo_%.bbappend

```
@@ -0,0 +1,3 @@
+do_install:append () {
+    echo "robot ALL=(ALL) ALL" >> ${D}${sysconfdir}/sudoers
```

Nit: could we use the sudoers.d directory
```suggestion
    echo "robot ALL=(ALL) ALL" >> ${D}${sysconfdir}/sudoers.d/robot
```